beta/nncf/tensorflow/quantization/layers.py

```diff
@@ -22,15 +22,16 @@
 
 @NNCF_CUSTOM_OBJECTS.register()
 class FakeQuantize(tf.keras.layers.Layer):
-    def __init__(self, config: TFQuantizerSpec, data_format='channels_last', **kwargs):
+    def __init__(self, config: TFQuantizerSpec, op_name: str, data_format: str ='channels_last', **kwargs):
```

Comment:
The op_name is not the required parameter of the FakeQuantize layer.

Reply:
I've change it, please check